### PR TITLE
Article page

### PR DESCRIPTION
**Pull request by AlduLonghi**
- Added a fixed positioned navigation bar.
- Embedded images and videos in the main part of the article.
- Added an absolutely positioned aside element with links.
- Used css grid layout technique to organize links to other articles at the bottom of the page.
- Added a sticky footer.

### CODE REVIEW

**Review by HADDADSOHAIB — CHANGES_REQUESTED** [review]
### Status: Change Required :recycle: 

### Hello @AlduLonghi and @Maryjanee  :hand: 

### Good job, you did great job with the project :+1: 

### Unfortunately, there are still some changes to get the project approved:
- You did not configure github actions correctly, please go to the following link and follow the instructions step by step. https://github.com/microverseinc/linters-config/tree/master/html-css
- Please fix the validators errors and warnings:
- [ ] Check HTML markup with this validator https://validator.w3.org/#validate_by_upload. (Students were asked to use this one, so in most cases their code should be already validated) ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/html_css/section/validation_requirement.png) from Pathwright)
Screenshot from 2020-06-15 10-33-10
- The navbar is not similar to the original:
- The original:
Screenshot from 2020-06-15 10-38-49
- Your Project:
Screenshot from 2020-06-15 10-39-36
- The social media icons should be links, both on the beginning and on the end of the articles:
Screenshot from 2020-06-15 10-41-20
- Editors picks articles should be links, I notice that you are using a link tag but the element is not acting as a link on the page, please fix that.
Screenshot from 2020-06-15 10-44-19
- There is a sticky footer on the original, please add that on your project:
Screenshot from 2020-06-15 10-49-53
### Once you are done with the changes, please ask for another code review. :+1:

File: index.html [hunk 1]
```diff
@@ -0,0 +1,867 @@
...
+        <a href="#" class="d-flex">
+          <img class="nav-icons" src="./assets/search.svg" alt="search-icon" />
+        </a>
+        <a href="#" class="nav-link">
+          Space & cosmos
+        </a>
+      </div>
+      <div class="logo-container">
+        <img class="nav-item logo" src="./assets/NYTLogo.svg" alt="NYT-logo" />
+      </div>
```
Comment (peterrobert) [inline index.html]: The logo should be a link. You can do this by adding an anchor tag

**Review by peterrobert — CHANGES_REQUESTED** [review]
### CHANGES REQUIRED :recycle: :stop_sign: 
Great work implementing the required changes by the previous TSE.
However, you need to improve the following issues before we approve your project

- [ ] Kindly add more description to your pull request message. Add bullet points explaining what you have done in this project

Also fix the issue below :arrow_down: 

If you have a question concerning the review kindly reach me via slack on @peterrobert

**HAPPY CODING**

**Review by BrittanyBlake — CHANGES_REQUESTED** [review]
### Status: Changes Required ♻️ 
****
### Hi @AlduLonghi  and @Maryjanee ! 👋 
You have done a great job implementing the changes requested by the previous TSE! 👍 Your project looks really good so far!🎖️🌟  There are just a few issues that need to be resolved before you can move on: ⬇️ 
****

- [ ] As requested by the previous TSE, the NY Times logo should be a link. Please see my comments below and fix this. 

- [ ]  I see that you have added `.DS_Store` to the` .gitignore` folder 👏 , However, there are still some `.DS_Store` files in your GitHub repository. Just adding it to the file does not remove the `.DS_Store` files that were previously added to Github. Please use this [Stack overflow discussion](https://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository) as a guide on how to remove the` .DS_Store` files.

Screen Shot 2020-06-16 at 1 19 11 AM
Screen Shot 2020-06-16 at 1 18 57 AM




****
### Please make the required changes and resubmit for a new code review once you are ready 🙏

Good Luck and Happy Coding 🍀 👩🏾‍💻

_P.S. If you have any questions about my review or need help, you can message me on slack (@ Brittany Blake)._

File: index.html (hunk 1 above)
Comment (BrittanyBlake) [reply index.html]: As requested by the previous TSE, please make the logo a link by adding an anchor tag

**Comment by Maryjanee** [comment]
- Converted Logo To a link.
- Removed .DS_Store from project repository

**Review by sumancrest0001 — APPROVED** [review]
**STATUS: APPROVED :100: :1st_place_medal: 
Congratulations TEAM :clap: :clap:** 
Your project looks good to me as you fulfilled all the minimum requirements from the Microverse curriculum and fixed issue mentioned by the previous TSE. Now, you can merge the branch to master and move ahead.

**Happy coding !!!**

**Pull request merged**